packages/client/components/ActivityLibrary/CreateNewActivity/CreateNewActivity.tsx

```diff
@@ -0,0 +1,263 @@
+import useRouter from '../../../hooks/useRouter'
+import PrimaryButton from '../../PrimaryButton'
+
+type ActivityType = 'retrospective' | 'poker'
+
+const SUPPORTED_CUSTOM_ACTIVITIES = [
+  {
+    title: 'Process Feedback',
+    type: 'retrospective' as ActivityType,
+    includedCategories: ['retrospective', 'feedback', 'strategy'],
+    image: newTemplate,
+    isEnabled: true,
+    badges: [
+      {category: 'retrospective', theme: 'bg-grape-500 text-white', title: 'Retrospective'},
+      {category: 'feedback', theme: 'bg-jade-400 text-white', title: 'Feedback'},
+      {category: 'strategy', theme: 'bg-rose-500 text-white', title: 'Strategy'}
+    ],
+    phases: [
+      {title: 'Add', description: 'comments'},
+      {title: 'Group', description: 'comments'},
+      {title: 'Discuss', description: 'topics'},
+      {title: 'Vote', description: 'on topics'},
+      {title: 'Create', description: 'takeaway tasks'}
+    ]
+  },
+  {
+    title: 'Estimate Items',
+    type: 'poker' as ActivityType,
+    includedCategories: ['estimation'],
+    image: estimatedEffortTemplate,
+    isEnabled: false,
+    badges: [{category: 'estimation', theme: 'bg-tomato-500 text-white', title: 'Esimation'}],
+    phases: [
+      {title: 'Select', description: 'or create issues to score'},
+      {title: 'Vote', description: 'on 1 or many scoring dimensions'},
+      {title: 'Push', description: 'esimations to your backlog'}
+    ]
+  }
+]
+
+const CategoryTitle = (props: ComponentPropsWithoutRef<'div'>) => {
+  const {children, className, ...rest} = props
+
+  return (
+    <div
+      className={clsx('p-4 text-lg font-semibold leading-5 text-slate-700', className)}
+      {...rest}
+    >
+      {children}
+    </div>
+  )
+}
+
+const query = graphql`
+  query CreateNewActivityQuery {
+    viewer {
+      featureFlags {
+        retrosInDisguise
+      }
+      teams {
+        id
+        tier
+        name
+        ...NewMeetingTeamPicker_selectedTeam
+        ...NewMeetingTeamPicker_teams
+      }
+      availableTemplates(first: 100) @connection(key: "ActivityLibrary_availableTemplates") {
+        edges {
+          node {
+            name
+            teamId
+          }
+        }
+      }
+    }
+  }
+`
+
+interface Props {
+  queryRef: PreloadedQuery<CreateNewActivityQuery>
+}
+
+export const CreateNewActivity = (props: Props) => {
+  const {queryRef} = props
+  const atmosphere = useAtmosphere()
+  const data = usePreloadedQuery<CreateNewActivityQuery>(query, queryRef)
+
+  const {match} = useRouter<{categoryId?: string}>()
+  const {params} = match
+
+  const [selectedActivity, setSelectedActivity] = useState(() => {
+    const defaultActivity = SUPPORTED_CUSTOM_ACTIVITIES[0]!
+    const categoryId = params.categoryId
+    if (!categoryId) return defaultActivity
+
+    const selectedActivity = SUPPORTED_CUSTOM_ACTIVITIES.find((category) =>
+      category.includedCategories.includes(categoryId)
```

Comment:
+1 the `category` here isn't so much a category as much as it's an activity type - as-is, the statement `category.includedCategories.includes(categoryId)` is kinda confusing, especially when you're finding the `selecterdActivity`

Reply:
you're 100% right!